docker/app/migrate_and_serve.sh

```diff
@@ -9,7 +9,12 @@
 
 echo [i] Applying migrations.
 python manage.py migrate --verbosity 1
+
 echo [i] Collecting static files.
 python manage.py collectstatic --no-input --clear --verbosity 0
+
+echo [i] Creating a superuser
```

Comment:
the other log lines here end with a dot and i feel like to stay consistent this one should, too